src/lib/utils/auto-prefixer.ts

```diff
@@ -10,24 +10,21 @@
 export function applyCssPrefixes(target) {
   for (let key in target) {
 
-    let value = target[key];
+    let value = target[key] || "";
+    let boxValue = toBoxValue(value);
+    let boxDirection = toBoxDirection(value);
```

Comment:
I think toBoxDirection and toBoxOrient will fail with non string parameters. I did a quick test with your changes and it fails with numeric values.